pkg/loop/cmd/loopinstall/README.md

```diff
@@ -44,11 +44,16 @@ plugins:
     - name: "default"
       moduleURI: "github.com/smartcontractkit/chainlink-cosmos"
       gitRef: "f740e9ae54e79762991bdaf8ad6b50363261c056"
-      installPath: "github.com/smartcontractkit/chainlink-cosmos/pkg/cosmos/cmd/chainlink-cosmos"
+      installPath: "./pkg/cosmos/cmd/chainlink-cosmos"
       libs:
         - "/go/pkg/mod/github.com/!cosm!wasm/wasmvm@v*/internal/api/libwasmvm.*.so"
 ```
 
+The `installPath` is relative to the root of the downloaded module but it can also be an absolute path like: `github.com/smartcontractkit/chainlink-cosmos/pkg/cosmos/cmd/chainlink-cosmos`. The absolute path will be stripped to the relative path (it is supported for backwards compatibility). The path is cleaned using Go's `filepath.Clean` function before use. This process normalizes the path by:
+
+- Resolving `.` (current directory) and `..` (parent directory) components (e.g., `path/to/../plugin` becomes `path/plugin`).
```

Comment:
Does the `../` path prefix, work? I don't see a test for it.

Reply:
Whoops, old comment. Thanks.

Updated in e278fadd